strapi/strapi
# feat(content-releases): add stage check to releases details page

### What does it do?

This PR refactors the Status column on Releases details page to add a Popover where we show both: fields validations & stage validation of each entry inside a release.

### How to test it?

1. Create a release and a Review Workflow with a required stage to publish
2. Add multiple entries to your release.
3. Check the status column is showing the expected information

### Known issue

- Icon colors are not okay. I will add a following PR on the DS for this. 
<img width="290" alt="Screenshot 2024-09-27 at 13 53 43" src="https://github.com/user-attachments/assets/21229ec7-47a1-4407-aca2-421d7a967fed">
